content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -42,11 +43,31 @@ const NewTabPage = React.createClass({
       this.setState({renderedOnce: true});
     }
   },
+  _hasWeightedHighligts() {
+    return this.props.WeightedHighlights.weightedHighlights && this.props.WeightedHighlights.rows.length;
```

Comment:
oops, you're right!

Let's actually move this to the NewTabPage selector, as we discussed